tools/cldr-code/src/main/java/org/unicode/cldr/util/XMLSource.java

```diff
@@ -1878,6 +1879,18 @@ public boolean equalsOrInheritsCurrentValue(String value, String curValue, Strin
         return false;
     }
 
+    /**
+     * Enable SourceLocation in this Source. Call this once before populating the XMLSource.
```

Comment:
I'm guessing that these must be overridden to make sense. Is that true?

Reply:
Yes, and they are overridden in SimpleXMLSource